remote-working-etiquette.md

```diff
@@ -0,0 +1,30 @@
+# Remote working etiquette
+
+Working remotely is a great way to get focussed, balance work and life, or maybe just have a change of scenery and be inspired. However, it's really important that while we enjoy these benefits and seek our own state of super-productivity, we remain conscious of the effectiveness of our colleagues and the team around us 
+
+## Remote considerations
+
+There is no list of acceptable reasons for when it's "ok" to work remotely. You'll be the best judge of whether not being physically present/absent is going to positively or negatively impact the success of the team. Here are some questions you might want to factor in to that decision:
+
+Environment
+- Will I be able to have good virtual access to the team from my remote location (i.e. internet speed able to support a Google Hangout)?
+- Will my environment allow me to focus on my work (i.e. minimal distraction from media, people and of course pets)?
+- Will my environment allow me to communicate easily with colleagues (i.e. minimal background noise and the ability to speak clearly/openly)?
+- Will I be able to run all systems (i.e VM) necessary to complete my work and support the team?
+
+Commitment
+- Will you still be able to make your commitment to any meetings?
+- Will working remotely hinder your ability to hit your sprint commitments?
+
+We all have various degrees of "cost" to being in the office, especially as we grow the team wider. It may be that you can peak personal productivity by saving your commute, immersing in the zen of working from the garden, being home to welcome a package so you don't have the chase around depots in the evening...
+
+## The key question
+
+Once these questions have been answered, you'll need to ask yourself: 
+
+Is the **benefit to the team** of me working remotely *greater than* the **impact on the team** of not being physically present.
```

Comment:
I think rather than suggesting that working in core office is by default a 'better' option the intention is to suggest benefits and disadvantages of either working location in a particular instance when we each plan our time in advance. This definitely not detract from the Anytime+ policy on my.hx but could help us to consider at which times working solo remotely could present the most benefits and when being co-located could offer the most benefits.